lib/ReactViews/NowViewing/Voldemort.jsx

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+import ObserveModelMixin from './../ObserveModelMixin';
+import React from 'react';
+
+const Voldemort = React.createClass({
+    mixins: [ObserveModelMixin],
+
+    propTypes: {
+        nowViewingItem: React.PropTypes.object
+    },
+
+    toggleOpen(item) {
+        item.isOpen = !item.isOpen;
+    },
+
+    toggleActive(item) {
+        item.isActive = !item.isActive;
+    },
+
+    renderVoldemortChildren(item, key) {
+        let toggleButton = null;
+        let selectButton = null;
+        const classNames = 'voldemort__children ' + (item.isOpen ? 'is-open' : '') + ' ' + (item.hasChildren ? 'has-children' : '');
+
+        if(item.hasChildren) {
+            toggleButton = <button onClick={this.toggleOpen.bind(this, item)} className={'btn btn--toggle ' + (item.isOpen ? 'is-open' : '')}></button>;
+        }
+        if(item.isSelectable) {
+            selectButton = <button onClick={this.toggleActive.bind(this, item)} className={'btn ' + (item.isActive ? 'btn--voldemort-active' : 'btn--voldemort-inactive')}></button>;
+        }
+
+        if(item.hasChildren) {
+            return (<li className={classNames}
+                        key={key}>
+                        {selectButton}
+                        {item.name}
+                        {toggleButton}
+                        <ul>
+                            {item.items.map((child, i)=>
+                                this.renderVoldemortChildren(child, i)
+                            )}
+                        </ul>
+                    </li>);
+        }
+        return <li className={classNames}
+                   key={key}>
+                   {selectButton}
+                   {item.name}
+                   {toggleButton}
+               </li>;
+    },
+
+    renderVoldemort() {
+        const nowViewingItem = this.props.nowViewingItem;
+        let content;
+
+        if(nowViewingItem.concepts && nowViewingItem.concepts.length > 0) {
+            content = nowViewingItem.concepts.map((item, i)=>{
+                if(item.name && item.isVisible) {
+                    return this.renderVoldemortChildren(item, i);
+                }
+            });
+        } else {
+            content = <li> empty </li>;
+        }
+        return content;
+    },
+
+    render() {
+        return (<div className='voldemort'>
+                  <ul>{this.renderVoldemort()}</ul>
+                </div>);
+    }
+});
+module.exports = Voldemort;
```

Comment:
I notice you're using radio buttons regardless of whether > 1 can be selected. Can you use checkboxes instead?
AbsConcepts (which are the top-level concepts for ABS catalog items) have an `isUnique` property that tells you if only 1 can be selected. DisplayVariableConcepts (which are the top-level concepts for csv files) have an `allowMultiple` property that tells you > 1 can be selected.  Hmmm... that's crazy. Let me rationalise this first.